stdlib/public/stubs/LibcShims.cpp

```diff
@@ -308,3 +318,67 @@ swift::_stdlib_cxx11_mt19937_uniform(__swift_uint32_t upper_bound) {
   std::uniform_int_distribution<__swift_uint32_t> RandomUniform(0, upper_bound);
   return RandomUniform(getGlobalMT19937());
 }
+
```

Comment:
You could add a reminder here to update the default `RandomNumberGenerator` documentation (if `_stdlib_random` is changed).